modules/nw-sriov-operator-uninstall.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/uninstalling-sriov-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-sriov-operator-uninstall_{context}"]
+= Uninstalling the SR-IOV Network Opreator
+
+As a cluster administrator, you can uninstall the SR-IOV Network Operator.
+
+.Prerequisites
+
+* You must have access to an {product-title} cluster using an account with `cluster-admin` permissions.
```

Comment:
I'd suggest removing 'must' to be more parallel with the next prereq's wording

Reply:
Fixed